ccache on centos is broken

cf. https://gitlab.icp.uni-stuttgart.de/espressomd/espresso/-/jobs/66484:
```
[  1%] Building CXX object src/config/CMakeFiles/EspressoConfig.dir/config-features.cpp.o
ccache: error: Failed to create temporary file for /cache/tmp/tmp.cpp_stderr: Permission denied
make[2]: *** [src/config/CMakeFiles/EspressoConfig.dir/config-features.cpp.o] Error 1
```
could be a side effect of #56 ?

--- a/docker/centos/Dockerfile-7
+++ b/docker/centos/Dockerfile-7
@@ -21,6 +21,6 @@ RUN yum -y install epel-release && yum -y install \
   && yum clean all \
   && ln -s /usr/bin/cmake3 /usr/bin/cmake \
   && pip2 install --upgrade pip; pip2 install cython pylint
-RUN useradd -m espresso
+RUN useradd -m espresso -u 1000
 USER 1000
 WORKDIR /home/espresso

--- a/docker/centos/Dockerfile-7
+++ b/docker/centos/Dockerfile-7
@@ -22,5 +22,5 @@ RUN yum -y install epel-release && yum -y install \
   && ln -s /usr/bin/cmake3 /usr/bin/cmake \
   && pip2 install --upgrade pip; pip2 install cython pylint
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/centos/Dockerfile-next
+++ b/docker/centos/Dockerfile-next
@@ -20,5 +20,5 @@ RUN yum -y install \
   ccache \
   && yum clean all
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/clang/Dockerfile-4.0
+++ b/docker/clang/Dockerfile-4.0
@@ -40,5 +40,5 @@ COPY boost.patch /tmp/
 RUN cd /usr/include/boost && patch -p1 < /tmp/boost.patch
 
 RUN useradd -m espresso && usermod -a -G www-data espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/clang/Dockerfile-6.0
+++ b/docker/clang/Dockerfile-6.0
@@ -52,5 +52,5 @@ COPY boost.patch /tmp/
 RUN cd /usr/include/boost && patch -p1 < /tmp/boost.patch
 
 RUN useradd -m espresso && usermod -a -G www-data espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/cuda/Dockerfile-8.0
+++ b/docker/cuda/Dockerfile-8.0
@@ -29,5 +29,5 @@ COPY thrust.patch /tmp/
 RUN cd /usr/local/cuda/include/thrust && patch -p1 < /tmp/thrust.patch
 
 RUN useradd -m espresso && usermod -a -G www-data espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/cuda/Dockerfile-9.0
+++ b/docker/cuda/Dockerfile-9.0
@@ -31,5 +31,5 @@ RUN apt-get update && apt-get install -y \
 && rm -rf /var/lib/apt/lists/*
 
 RUN useradd -m espresso && usermod -a -G www-data espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/debian/Dockerfile-9
+++ b/docker/debian/Dockerfile-9
@@ -22,6 +22,6 @@ RUN apt-get update && apt-get install -y \
 && rm -rf /var/lib/apt/lists/*
 
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso
 

--- a/docker/intel/Dockerfile-17
+++ b/docker/intel/Dockerfile-17
@@ -65,5 +65,5 @@ RUN cd /tmp \
 ENV BOOST_ROOT=/opt/boost LD_LIBRARY_PATH="/opt/boost/lib:${LD_LIBRARY_PATH}"
 
 RUN useradd -m espresso && usermod -a -G www-data espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/opensuse/Dockerfile-15.0
+++ b/docker/opensuse/Dockerfile-15.0
@@ -19,5 +19,5 @@ RUN zypper -n --gpg-auto-import-keys refresh \
   curl \
   && update-alternatives --install /usr/bin/pycodestyle pycodestyle /usr/bin/pycodestyle-2.7 1
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/opensuse/Dockerfile-42.3
+++ b/docker/opensuse/Dockerfile-42.3
@@ -19,5 +19,5 @@ RUN zypper -n --gpg-auto-import-keys refresh \
   curl
 RUN pip2 install --upgrade pip && pip2 install pylint --upgrade
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/rocm/Dockerfile-latest
+++ b/docker/rocm/Dockerfile-latest
@@ -27,6 +27,6 @@ RUN apt-get update && apt-get install -y \
 && rm -rf /var/lib/apt/lists/*
 
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso
 

--- a/docker/ubuntu-python3/Dockerfile-16.04
+++ b/docker/ubuntu-python3/Dockerfile-16.04
@@ -21,6 +21,6 @@ RUN apt-get update && apt-get install -y \
 && rm -rf /var/lib/apt/lists/*
 
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso
 

--- a/docker/ubuntu-python3/Dockerfile-18.04
+++ b/docker/ubuntu-python3/Dockerfile-18.04
@@ -23,6 +23,6 @@ RUN apt-get update && apt-get install -y \
 && rm -rf /var/lib/apt/lists/*
 
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso
 

--- a/docker/ubuntu/Dockerfile-16.04
+++ b/docker/ubuntu/Dockerfile-16.04
@@ -31,5 +31,5 @@ RUN apt-get update && apt-get install -y \
 RUN bash /tmp/build-and-install-scafacos.sh
 
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/ubuntu/Dockerfile-18.04
+++ b/docker/ubuntu/Dockerfile-18.04
@@ -30,7 +30,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
 RUN bash /tmp/build-and-install-scafacos.sh
 
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso
 ENV OMPI_MCA_plm=isolated
 

--- a/docker/ubuntu/Dockerfile-summerschool
+++ b/docker/ubuntu/Dockerfile-summerschool
@@ -13,5 +13,5 @@ RUN apt-get update && apt-get install -y \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
 
-USER espresso
+USER 1000
 WORKDIR /home/espresso

--- a/docker/ubuntu/Dockerfile-wo-dependencies
+++ b/docker/ubuntu/Dockerfile-wo-dependencies
@@ -19,5 +19,5 @@ RUN apt-get update && apt-get install -y \
 && rm -rf /var/lib/apt/lists/*
 
 RUN useradd -m espresso
-USER espresso
+USER 1000
 WORKDIR /home/espresso

